src/cmd/create.go

```diff
@@ -646,6 +647,7 @@ func getServiceSocket(serviceName string, unitName string) (string, error) {
 }
 
 func pullImage(image, release string) (bool, error) {
+	logrus.Debugf("Looking for image %s", image)
```

Comment:
Nitpick: a newline after the `logrus.Debugf` will be good because it will separate the log message from the actual implementation of the function.